doc/cephadm/mgr.rst

```diff
@@ -0,0 +1,36 @@
+.. _mgr-cephadm-mgr:
+
+MGR Service
+===========
+
+The cephadm MGR service is hosting different modules, like the :ref:`mgr-dashboard`
+and the cephadm manager module.
+
+.. _cephadm-mgr-networks:
+
+Specifying Networks
+-------------------
+
+The MGR service supports binding only to a specific IP within a network.
+
+example spec file (leveraging a default placement):
+
+.. code-block:: yaml
+
+    service_type: mgr
+    networks:
+    - 192.169.142.0/24
+
+Allow co-location of MGR daemons
+================================
```

Comment:
When I checked the docs generated for this by read the docs this "Allow co-location of MGR daemons" and the "Further Reading" Section were getting their own spot in the sidebar 
![image](https://user-images.githubusercontent.com/47704447/134336671-74f97c16-0c41-4faa-ae50-2b2f66d5af6d.png)

I think it's because of the double line of dashes, as opposed to using a single line like you have for "Specifying Networks" that makes it a subsection